graph/src/data/store/mod.rs

```diff
@@ -114,6 +119,55 @@ impl<'a> From<&'a String> for Value {
     }
 }
 
+impl ToSql<Bool, Pg> for Value {
+    fn to_sql<W: Write>(&self, out: &mut Output<W, Pg>) -> serialize::Result {
+        match *self {
+            Value::Bool(ref b) => <bool as ToSql<Bool, Pg>>::to_sql(&b, out),
+            _ => unimplemented!(),
```

Comment:
`unimplemented` isn't the right description here, a custom panic message would be better.